utils/constants.ts

```diff
@@ -36,7 +40,7 @@ export const URLS = {
     nftStorage: 'https://nft-storage.kodadot.workers.dev/',
     netlify: 'https://beta.kodadot.xyz/.netlify/functions/',
     seoCard: 'https://og-image-green-seven.vercel.app/',
-    rubick: 'https://squid.subsquid.io/rubick/v/007/graphql',
+    rubick: 'https://squid.subsquid.io/rubick/v/008/graphql',
```

Comment:
wut? 008 isn't yet ready

Reply:
yep. 
#4633  needs to be merged first and them I'll resolve conflicts